### PR TITLE
[14.0][FIX] product_contract: fixes the visibility of the is_contract field in the sales order form

### PR DESCRIPTION
**Pull request by marcelsavegnago**

### CODE REVIEW

**Comment by marcelsavegnago** [comment]
ping @douglascstd @augustodinizl @aug1804 @kaynnan

**Review by douglascstd — APPROVED** [review]
[FUNCTIONAL TESTE] APPROVED

From:
image
image


TO:
image
image

File: product_contract/views/sale_order.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
                 </button>
             </xpath>
             <xpath expr="//field[@name='order_line']" position="before">
-                <field name="is_contract" attrs="{'invisible': []}" />
+                <field name="is_contract" attrs="{'invisible': True}" />
```
Comment (rousseldenis) [inline product_contract/views/sale_order.xml]: In this case use :


```suggestion
                <field name="is_contract" invisible="1" />
```

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

File: product_contract/views/sale_order.xml (hunk 1 above)
Comment (marcelsavegnago) [reply product_contract/views/sale_order.xml]: @rousseldenis  Done

**Review by rousseldenis — APPROVED** [review]

**Comment by marcelsavegnago** [comment]
ping @sbejaoui  @pedrobaeza

**Pull request merged**